sdk/communication/azure-communication-rooms/azure/communication/rooms/_models/_models.py

```diff
@@ -0,0 +1,224 @@
+# coding=utf-8
+# --------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------
+from datetime import datetime
+from typing import List, Optional, Union
+
+from azure.core.exceptions import DeserializationError
+from .._generated.models import (
+    RoomParticipant as RoomParticipantInternal,
+    CommunicationIdentifierModel,
+    CommunicationUserIdentifierModel,
+    RoomJoinPolicy,
+    RoleType
+)
+from .._generated import _serialization
+from .._generated.models import RoomModel
+from .._shared.models import (
+    CommunicationIdentifier,
+    CommunicationIdentifierKind,
+    CommunicationUserIdentifier,
+    UnknownIdentifier
+)
+
+
+class CommunicationRoom(_serialization.Model):
```

Comment:
Is this model generated? Or hand built?
We shouldn't have non-generated models that extend _serialization.Model.